areaOfInterest/aoi-demo-nopics.html

```diff
@@ -0,0 +1,30 @@
+<html class="no-js" lang="en" dir="ltr">
```

Comment:
You can remove this file, there's a copy in the ramp repo.

Reply:
Done